### PR TITLE
Adjust touchpad edge detection for X1Tab3

### PR DESCRIPTION
**Pull request by amosbird**
Results are retrieved from `touchpad-edge-detector`

### CODE REVIEW

File: hwdb.d/60-evdev.hwdb [hunk 1]
```diff
@@ -434,6 +434,13 @@ evdev:name:Synaptics TM3289-002:dmi:*svnLENOVO*:pvrThinkPadX1Carbon5th*
  EVDEV_ABS_35=::19
  EVDEV_ABS_36=::19
 
+# Lenove Thinkpad X1 Tablet Gen3.
+evdev:input:b0003v17EFp60B5*
+ EVDEV_ABS_00=0:1163:12
```
Comment (whot) [inline hwdb.d/60-evdev.hwdb]: judging by the pastebin output I saw earlier for this device, the axis ranges are correct, it's just the resolution that is off. Let's use the shortened format like in the entry above where `0:1163:12` becomes just `::12`. Same with the other axes please.

File: hwdb.d/60-evdev.hwdb [hunk 2]
```diff
@@ -434,6 +434,13 @@ evdev:name:Synaptics TM3289-002:dmi:*svnLENOVO*:pvrThinkPadX1Carbon5th*
  EVDEV_ABS_35=::19
  EVDEV_ABS_36=::19
 
+# Lenove Thinkpad X1 Tablet Gen3.
```
Comment (whot) [inline hwdb.d/60-evdev.hwdb]: typo: "lenove". And no full stop please

Comment (amosbird) [reply hwdb.d/60-evdev.hwdb]: fixed. Other comments also contain full stop...

File: hwdb.d/60-evdev.hwdb (hunk 1 above)
Comment (amosbird) [reply hwdb.d/60-evdev.hwdb]: Done

File: hwdb.d/60-evdev.hwdb (hunk 2 above)
Comment (whot) [reply hwdb.d/60-evdev.hwdb]: looks like the ones that contain the full stop are the ones that end in `gen` which is obvs short for `generation` but that doesn't apply here with `gen3`. And `gen. 3` makes less sense, so getting rid of it makes the most sense IMO.

Comment (amosbird) [reply hwdb.d/60-evdev.hwdb]: Ok...  Fixed.

**Pull request merged**